pmm-app/tests/pages/dashboardPage.js

```diff
@@ -307,6 +307,25 @@ module.exports = {
     otherReportTitleWithNoData:
       "//span[contains(text(),'No Data')]//ancestor::div[contains(@class,'panel-container')]//span[contains(@class,'panel-title-text')]",
     collapsedDashboardRow: "//div[@class='dashboard-row dashboard-row--collapsed']/a",
+    annotationMarker: "(//div[contains(@class,'events_marker')])",
+  },
+
+  annotationLocator(annotationNumber) {
+    return "(//div[contains(@class,'events_marker')])[" + annotationNumber + "]";
```

Comment:
Many find that lots of quotes in one line are rather difficult to read and comprehend. I'd suggest to use template literals instead:
```suggestion
    return `(//div[contains(@class,'events_marker')])[${annotationNumber}]`;
```